global/js/comment_system.js

```diff
@@ -359,9 +370,20 @@ function openReply(strCommentId, element) {
 
         document.getElementById("reply__send").addEventListener("click", () => {
             if (document.getElementById("reply__input").value.includes("https://osu.ppy.sh/beatmapsets/")) {
-                openDialog("Are you sure you want to post this comment?", "Your reply looks like it contains an osu! beatmap URL.", "If you're trying to post a beatmap, you should instead use the 'Add' button on the beatmaps panel.", "Post Anyway", function () {
-                    replySend();
-                });
+                openDialog("Are you sure you want to post this comment?", "Your reply looks like it contains an osu! beatmap URL.", "If you're trying to post a beatmap, you should instead use the 'Add' button on the beatmaps panel.", [

```

Comment:
Would this be replaced with a translation key?

Reply:
nah, would just be passed as string from a function, best to do that if possible in most places